README.md

```diff
@@ -104,23 +104,23 @@ The table below shows the time it takes to generate a proof for a hash chain of
     </thead>
     <tbody>
         <tr>
-            <td colspan=8>Apple M2 (4P + 4E cores), 8GB RAM </td>
+            <td colspan=8>Apple M2 (4P + 4E cores), 24GB RAM </td>
```

Comment:
I think this needs to be updated on line 181 as well.

Reply:
you are right, thanks